apis/kueue/v1beta1/workload_types.go

```diff
@@ -386,12 +386,15 @@ type RequeueState struct {
 	RequeueAt *metav1.Time `json:"requeueAt,omitempty"`
 }
 
+// AdmissionCheckReference is the name of an AdmissionCheck.
+// +kubebuilder:validation:Required
+// +kubebuilder:validation:MaxLength=316
+type AdmissionCheckReference string
+
 type AdmissionCheckState struct {
-	// name identifies the admission check.
+	// Name identifies the admission check.
```

Comment:
```suggestion
	// name identifies the admission check.
```

API documents should use YAML field name since this is user-facing.